introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -26,7 +26,7 @@
     "---\n",
```

Comment:
<ol><li>Leaving a note to make sure we don't forget to change alpha bucket name.</li><li>Can <code>s3://</code>  schema be used instead of <code>https://</code>?</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1713/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yes. I won't revise this or any other notebook till I have fixed the bucket. Also, have added a note in all PRs asking reviewers to not merge the code.</p>